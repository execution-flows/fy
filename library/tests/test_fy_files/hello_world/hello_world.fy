flow HelloWorld:
    def -> None:
        print("Hello world!")
