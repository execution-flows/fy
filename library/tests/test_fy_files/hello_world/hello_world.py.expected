from base.execution_flow_base import ExecutionFlowBase


class HelloWorld_Flow(
    # Base
    ExecutionFlowBase[None]
):
    def __call__(self) -> None:
        print("Hello world!")
